Rollbar/Rollbar.h

```diff
@@ -7,7 +7,17 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "RollbarKSCrashInstallation.h"
```

Comment:
Do we need this import here?

Reply:
You are right, this is not necessary, removed.